localstack/utils/collections.py

```diff
@@ -533,3 +533,18 @@ def is_comma_delimited_list(string: str, item_regex: Optional[str] = None) -> bo
     if pattern.match(string) is None:
         return False
     return True
+
+
+def parse_key_value_pairs(raw_text: str) -> dict:
```

Comment:
nit: could add a docstring here with for example the values from one of the test cases below to clearly demonstrate its purpose.

Reply:
Thanks for the suggestion!